lib/valkyrie/derivative_service.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+require 'hydra/derivatives'
+module Valkyrie
+  class DerivativeService
+    Hydra::Derivatives.source_file_service = Valkyrie::LocalFileService
+    Hydra::Derivatives.output_file_service = Valkyrie::PersistDerivatives
```

Comment:
Are we reading source files from local files and writing the derivs to local files because that's the only file storage option?  How does this work when we have more options?